README.md

```diff
@@ -7,10 +7,10 @@ delete, and enumerate
 resources to be managed by Chef.
 
 ## Installation
-Be sure you are running the latest version of Chef, which can be installed
+Be sure you are running the latest version of Chef DK, which can be installed
 via:
 
-    gem install chef
+    https://downloads.chef.io/chef-dk/
 
 This plugin is distributed as a Ruby Gem. To install it, run:
```

Comment:
:+1: to the comment about updating the installation command to `chef gem install knife-azure`. I haven't found any dependency conflicts with installing knife-azure into the chef-dk, what conflicts are you referring to @siddheshwar-more?